test/tab-components/tabTest.js

```diff
@@ -89,6 +91,18 @@ describe('tab tests', function () {
         .waitForExist('[data-test-id="tab"][data-frame-key="2"]')
         .waitForTextValue('[data-test-id="tab"][data-frame-key="2"]', 'New Tab')
     })
+
+    it('shows empty urlbar', function * () {
```

Comment:
both of these tests pass even when I remove the urlbar changes so I'm not sure if they really capture the issue